### PR TITLE
sap_vm_provision: exec flow amendments from testing

### PR DESCRIPTION
**Pull request by sean-freeman**
sap_vm_provision: exec flow amendments from testing

### CODE REVIEW

**Review by marcelmamula — APPROVED** [review]
LGTM after connectivity typo is resolved @sean-freeman I suppose removal of AWS and GCP from Readme is because Ansible collections are used instead of CLI, right?

**Comment by berndfinger** [comment]
> LGTM after connectivity typo is resolved @sean-freeman I suppose removal of AWS and GCP from Readme is because Ansible collections are used instead of CLI, right?

@marcelmamula As there has been no reply yet from @sean-freeman, should we just assume that the answer to your question is `yes`, and just merge and fix the spelling error and the ansible-lint error in a new small PR right after merging?

File: roles/sap_vm_provision/tasks/platform_ansible/ibmcloud_powervs/execute_main.yml [hunk 1]
```diff
@@ -358,8 +368,13 @@
...
         line: nameserver 161.26.0.10
 
-    # Not applicable to the IBM PowerVS Workspace enabled with Power Edge Router (from Q4-2023 onwards)
-    - name: Register Web Forward Proxy (for legacy Power Infrastructure Cloud Connections networking configuration)
+    # Required Web Forward Proxy
+    # For IBM PowerVS Workspace enabled with Power Edge Router (from Q4-2023 onwards),
+    # the SNAT (VPC Public Gateway) service is not routable from IBM Power Virtual Server hosts
+    # and cannot be used for outbound Public Internet connectivity
+    # For IBM PowerVS Workspace with legacy Cloud Connection,
+    # required for both outbound Public Internet connectivty and internal traffic to other IBM Cloud Services
```
Comment (Wabri) [inline roles/sap_vm_provision/tasks/platform_ansible/ibmcloud_powervs/execute_main.yml:376]:
```suggestion
    # required for both outbound Public Internet connectivity and internal traffic to other IBM Cloud Services
```

File: roles/sap_vm_provision/tasks/platform_ansible/ibmcloud_vs/execute_setup_ha.yml [hunk 2]
```diff
@@ -1014,235 +1246,6 @@
...
-  ibm.cloudcollection.ibm_is_lb_listener:
-    lb: "{{ (ibmcloud_lbs_all_info | selectattr('name', '==', sap_vm_provision_ha_load_balancer_name_nwas + '-ers'))[0].id }}"
-    default_pool: "{{ (__sap_vm_provision_task_ibmcloud_lb_pools.results | json_query('[*].resource.pools') | flatten | selectattr('name', '==', sap_vm_provision_ha_load_balancer_name_nwas + '-ers-pool-sapctrls'))[0].id }}"
-    protocol: tcp
-    port: "{{ ('5' + sap_system_nwas_abap_ers_instance_nr + '14') | int }}"
-    idle_connection_timeout: 600 # 10 minutes
-    ibmcloud_api_key: "{{ sap_vm_provision_ibmcloud_api_key }}"
-  when: (groups['nwas_ers'] is defined and (groups['nwas_ers'] | length>0))
-  failed_when: not __sap_vm_provision_task_ibmcloud_lb_frontend_listener_ers5.rc == 0 and not 'listener_duplicate_port' in __sap_vm_provision_task_ibmcloud_lb_frontend_listener_ers5.stderr
-
```
Comment (Wabri) [inline roles/sap_vm_provision/tasks/platform_ansible/ibmcloud_vs/execute_setup_ha.yml:1249]:
```suggestion
```

Too many blank line => https://github.com/sap-linuxlab/community.sap_infrastructure/actions/runs/10631453083/job/29472374510?pr=58

**Review by Wabri — CHANGES_REQUESTED** [review]
Minor linting problem to fix

**Pull request merged**